plugins/google-oauth2-before.conf

```diff
@@ -51,9 +51,13 @@ SecRule TX:GOOGLE-OAUTH2-PLUGIN_CALLBACK_DETECTED "@eq 1" \
     t:none,\
     nolog,\
     ver:'google-oauth2-plugin/1.0.0',\
-    chain
-    SecRule ARGS_NAMES "!@pm state code scope authuser hd prompt" \
-        "setvar:'tx.google-oauth2-plugin_callback_detected=0'"
+    chain"
+    SecRule ARGS_NAMES "@rx ^.*$" \
```

Comment:
You could use `@pmFromFile` instead, then users could add exceptions to a file. Just an idea. If you think this if fine, then it's ok.

Reply:
I choosed this method because it is used also in other parts of CRS.